cmd/miner/client.go

```diff
@@ -131,7 +131,7 @@ func (m *Miner) keepAlive(ctx context.Context) {
 			m.encoder = json.NewEncoder(m.conn)
 			m.reader = bufio.NewReader(m.conn)
 
-			log.Debugf("miner connected to %s", m.config.Pool)
+			log.Tracef("miner connected to %s", m.config.Pool)
```

Comment:
I dont think this needs to be relegated to trace, its fine on debug. Tbh I might  even argue that this line would be fine at info level.

Trace should be reserved for things which either log excessive amount of data, or are logged very frequently. This line does neither.